tests/tests.js

```diff
@@ -2,6 +2,14 @@
 
 navigator.serviceWorker.register('service-worker.js');
 
+var controlledPromise = new Promise(function(resolve) {
```

Comment:
Could you add a comment here.

I understand the reasoning behind this - but its still hazy and others may not be aware of the reasons.